Shear/History.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  History.swift
+//  Shear
+//
+//  Created by Andreas Kompanez on 23.02.20.
+//  Copyright © 2020 Andreas Kompanez. All rights reserved.
+//
+
+import Foundation
+
+enum HistoryStoreError: Error {
+    case undefined
+}
+
+typealias HistoryStoreHandler = (_ error: HistoryStoreError?, _ item: HistoryItem) -> Void
+
+class History: HistoryType {
+    let storage: HistoryStorageType
+    
+    init(storage: HistoryStorageType) {
+        self.storage = storage
+    }
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Shear/blob/895e50bbb85826ea77037e4a1c1e5468d960c4fc/Shear/History.swift#L23"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>History.swift:23</code></del></td>
    </tr>
  </tbody>
</table>